adviser/overlays/ocp4-stage/configmap.yaml

```diff
@@ -0,0 +1,9 @@
+---
+kind: ConfigMap
```

Comment:
we would need to add this to the kustomize file as well.
please keep this under the patchstrategymerge:
https://github.com/thoth-station/thoth-application/blob/a115c78b4a32841e7f9e259a45bc4b2232174d08/adviser/overlays/ocp4-stage/kustomization.yaml#L10

Reply:
Done in 1d1b34d0, thanks for spotting this 👍🏻